setup.go

```diff
@@ -14,7 +14,7 @@ import (
 )
 
 const (
-	downloadURL   = "https://get.docker.com/builds/Linux/x86_64/docker-"
+	downloadURL   = "https://download.docker.com/linux/static/stable/x86_64/docker-"
```

Comment:
Also wondering if the whole version-compare in `GetDockerBinary()` should still be there; versions older than Docker-CE 17.09 have reached EOL, so perhaps it should be simplified to remove that code.